CoreScriptsRoot/Modules/DeveloperConsoleModule.lua

```diff
@@ -273,11 +273,15 @@ end
 
 function Methods.ResetFrameDimensions(devConsole)
 	devConsole.Frame.Size = UDim2_new(0.5, 20, 0.5, 20);
-	devConsole.Frame.Position = UDim2_new(0.25, -10, 0.125, -10)
+	
+	local abSize = devConsole.Frame.AbsoluteSize
+	devConsole:SetFrameSize(abSize.x, abSize.y)
```

Comment:
I'm sure there is a reason for this, but it seems weird. Could you explain the reasoning here? maybe we should comment it